watertap/property_models/air_water_equilibrium_prop_pack.py

```diff
@@ -0,0 +1,2695 @@
+            self.params.volatile_comps, expr=expr_collision_function_zeta_comp
+        )
+
+    def _collision_function_ee_comp(self):
+        def expr_collision_function_ee_comp(b, j):
+            t = b.temperature["Vap"]
+            return log10((boltzmann * t) / b.energy_molecular_attraction[j])
+
+        self.collision_function_ee_comp = Expression(
+            self.params.volatile_comps, expr=expr_collision_function_ee_comp
+        )
+
+    def _molar_volume_comp(self):
+
+        if (
+            self.params.config.molar_volume_calculation
+            == MolarVolumeCalculation.TynCalus
+        ):
+            self.molar_volume_comp = Var(
+                self.params.volatile_comps,
+                initialize=self.params.config.molar_volume_data,
+                units=pyunits.m**3 / pyunits.mol,
+                doc="Molar volume of solutes",
+            )
+
+            def rule_molar_volume_comp(b, j):
+                return b.molar_volume_comp[j] == pyunits.convert(
+                    b.params.tyn_calus_param
+                    * pyunits.convert(
+                        b.molar_volume_comp_crit[j],
+                        to_units=pyunits.cm**3 / pyunits.mol,
+                    )
+                    ** b.params.tyn_calus_exponent,
+                    to_units=pyunits.m**3 / pyunits.mol,
+                )
+
+            self.eq_molar_volume_comp = Constraint(
+                self.params.volatile_comps, rule=rule_molar_volume_comp
+            )
+
+        else:
+            add_object_reference(
+                self, "molar_volume_comp", self.params.molar_volume_comp
+            )
+
+    def _henry_comp(self):
+        if self.params.config.temp_adjust_henry:
+
+            add_object_reference(
+                self, "henry_comp_ref", self.params.henry_comp
+            )  # assume the data provided to config.henry_constant_data is @25C
+
+            self.henry_comp = Var(
+                self.params.volatile_comps,
+                initialize=self.params.config.henry_constant_data,
+                units=pyunits.dimensionless,
+                doc="Temperature adjusted dimensionless Henry's constant",
+            )
+
+            def rule_henry_comp(b, j):
+                t0 = 298 * pyunits.degK
+                exponential_term = pyunits.convert(
+                    (-b.enth_change_dissolution_comp[j] / Constants.gas_constant)
+                    * (1 / b.temperature["Liq"] - 1 / t0),
+                    to_units=pyunits.dimensionless,
+                )
+                return b.henry_comp[j] == b.henry_comp_ref[j] * exp(exponential_term)
+
+            self.eq_henry_comp = Constraint(
+                self.params.volatile_comps,
+                rule=rule_henry_comp,
+                doc="van't Hoff equation",
+            )
+
+        else:
+            add_object_reference(self, "henry_comp", self.params.henry_comp)
+
+    # pylint: disable=function-redefinition
```

Comment:
@lbianchi-lbl and here is my attempt at coercing pylint

Reply:
@ksbeattie ok putting it on each line and then running black caused it to not be on the same line anymore, so I put it at the top of the file, let's see if that works

Reply:
Heh, didn't work.  I think just putting the one disable at the beginning of the `_pressure_vap_sat function` (~line 2178) might be the best approach here.